server-common/src/main/java/com/datastrato/gravitino/server/auth/OAuthConfig.java

```diff
@@ -33,7 +33,7 @@ public interface OAuthConfig extends Configs {
           .doc("The signing key of JWT when Gravitino uses OAuth as the authenticator")
           .version("0.3.0")
           .stringConf()
-          .createWithDefault("");
+          .createWithNoDefault();
```

Comment:
Maybe you should check all the configs not only here, but also in each catalogs, to change them if there're no default value.

Reply:
OK. I will modify them at the same time.

Reply:
Fixed.